superset/views/core.py

```diff
@@ -2456,7 +2458,7 @@ def _sql_json_async(  # pylint: disable=too-many-arguments
         except Exception as ex:  # pylint: disable=broad-except
             logger.exception("Query %i: %s", query.id, str(ex))
 
-            message = _("Failed to start remote query on a worker.")
```

Comment:
Single quotes are lazy translations that can't be serialized to JSON; these need to be changed to the regular translation to be used in the SIP-40 error payload.

Reply:
I wonder if we can work on our json serializer instead and make them support lazy strings.